lib/handler/retry-handler.js

```diff
@@ -29,13 +40,13 @@ class RetryHandler {
       statusCodes
     } = retryOptions ?? {}
 
-    this.dispatch = handlers.dispatch
-    this.handler = handlers.handler
-    this.opts = dispatchOpts
-    this.abort = null
-    this.aborted = false
-    this.retryOpts = {
-      retry: retryFn ?? RetryHandler[kRetryHandlerDefaultRetry],
+    this.#opts = dispatchOpts // TODO (fix): This assumes that opts are not mutated...
+    this.#dispatch = dispatch
+    this.#handler = handler
```

Comment:
I would recommend not doing this, as functions could be in there too. Is there a need?

Reply:
Sorry, I don't understand.